CONTRIBUTING.md

```diff
@@ -17,3 +17,32 @@ rover is an open source project developed by [Apollo GraphQL](https://apollograp
 - One approval from a member of the Apollo tooling team.
   - The team consists of @jbaxleyiii, @ashleygwilliams, @JakeDawkins, and @EverlastingBugstopper.
   - PRs with changes in `docs/` automatically add @StephenBarlow as a reviewer. PRs are not blocked on this approval.
+
+## Adding new commands/subcommands
+
+When adding a new command or a subcommand, you should make sure that the field in the new command struct is titled `command`. This allows our automated anonymous usage collection to work properly.
+
+Should you add arguments that are _not_ subcommands, the arguments you specify will automatically be reported to Apollo's Telemetry backend, unless you specify that an argument should be ignored with `#[serde(skip_serializing)]`. You should be careful to add the `skip_serializing` attribute to any fields that may contain data that may not be anonymous. This can be anything from a profile name to a path on the user's computer that could include their name.
+
+If you would like to verify that your new command works with the usage data capturing system, you can spin up a development server at `http://localhost:8787/telemetry` (or, if you'd like to use some other endpoint you may specify `APOLLO_TELEMETRY_URL`) and make sure the `command` section of the request body looks right. One example of proper usage collection is when the command `rover config profile show default --sensitive` is run, the request body should look something like this:
```

Comment:
"If you would like to verify" -> "To verify"

(or, if you'd like...) -> (or specify a different endpoint by setting `APOLLO_TELEMETRY_URL`)

"looks right" -> "looks correct"

New paragraph for "One example..."

"One example..." -> "For example, the usage collection request body for the command `rover config profile show default --sensitive` looks like the following:"